cmake/StormSetup.cmake

```diff
@@ -224,7 +224,9 @@ macro(STORM_SETUP)
     foreach(module ${global_modules_list})
       if(TARGET ${module})
         # TODO: make it portable? (at least for gcc it is --whole-archive)
-        list(APPEND target_link_flags "/WHOLEARCHIVE:${module}")
+        if (WIN32)
```

Comment:
does it work without `-force_load` for clang?

Reply:
As I wrote in the description - the main goal is full Linux building + CI. This is minimal fix to get Linux linking, but proper fix will require 1 more change:
https://github.com/q4a/storm-engine/commit/8c6b63b55b5475fb5c3de9f6da5f2353f322c046
Should I include it in this PR or the next one?

Reply:
I added https://github.com/q4a/storm-engine/commit/8c6b63b55b5475fb5c3de9f6da5f2353f322c046 to this PR.